src/main/java/com/alexx/employeesAndDepartments/services/DepartmentService.java

```diff
@@ -50,6 +53,7 @@ public DepartmentDTO editDepartment(@Valid DepartmentDTO departmentDTO) {
             departmentEntity.setAdditionalInformation(departmentDTO.getAdditionalInformation());
         departmentRepository.save(departmentEntity);
         departmentDTO = modelMapper.map(departmentEntity, DepartmentDTO.class);
+        log.info("Department with ID {} edited", departmentDTO.getId());
```

Comment:
Please read about different level of logging (DEBUG, INFO, TRACE) and apply what you feel is needed

Reply:
ok